pkg/object/redis.go

```diff
@@ -69,6 +71,52 @@ func (r *redisStore) Delete(key string) error {
 	return r.rdb.Del(c, key).Err()
 }
 
+func (t *redisStore) List(prefix, marker string, limit int64) ([]Object, error) {
+	if marker == "" {
+		marker = prefix
+	}
+	if limit > 1000 {
+		limit = 1000
+	}
+	var keyList []string
+	var cursor uint64
+	for {
+		keys, c, err := t.rdb.Scan(context.TODO(), cursor, marker+"*", limit).Result()
```

Comment:
marker means the first key, not prefix

Reply:
the results should be sorted by key, so we should get implement the ListAll(), get all the keys, sort them, then return them